src/sql/src/parsers/copy_parser.rs

```diff
@@ -56,7 +56,7 @@ impl<'a> ParserContext<'a> {
             })?;
 
         let req = if self.parser.parse_keyword(Keyword::TO) {
-            let (with, connection, location) = self.parse_copy_parameters()?;
```

Comment:
Perhaps we should fail if COPY DATABASE with a LIMIT param? This seems an invalid input.